package.json

```diff
@@ -162,20 +163,21 @@
     "copy-webpack-plugin": "^4.0.1",
     "css-loader": "^0.28.4",
     "enzyme": "^2.9.1",
-    "extract-text-webpack-plugin": "^2.1.2",
+    "extract-text-webpack-plugin": "^3.0.0",
     "faker": "^4.1.0",
     "file-loader": "^0.11.2",
     "fs-extra": "^3.0.1",
     "git-revision-webpack-plugin": "^2.5.1",
     "graphql": "^0.10.3",
+    "graphql-api": "^0.1.7",
```

Comment:
@samcdavid no not exactly...the authoritative API still lives on the server but i use a lot of the GraphQL server tools to help build out a good mock environment for testing. 

See - https://github.com/verypossible/booster-kit-react/blob/chore/misc-cleanup/src/lib/test/mockApollo.tsx